cfscrape/__init__.py

```diff
@@ -66,6 +83,9 @@ class CloudflareCaptchaError(CloudflareError):
 
 
 class CloudflareScraper(Session):
+    # AES128-SHA ciphers seem to provoke a cloudflare challenge captcha.
+    captcha_adapter = CaptchaProvokingCiphersRemover(["AES128-SHA"])
```

Comment:
`requests` doesn't resuse the adapters when instantiating a new session. https://github.com/kennethreitz/requests/blob/master/requests/sessions.py#L416-L418

Reply:
Yeah, I'm currently wondering if it's even allowed to unmount the adapter, seeing that it holds a connection pool as well...